### ISSUE TITLE
Does kasmvnc support subdomain?

### ISSUE THREAD

**Issue by Davi-He** [coder/modules#361]
Does kasmvnc support subdomain? I want to use subdomain.

**Comment by matifali** [comment]
Hi @Davi-He do you mean to run it without a subdomain? It runs by default on a subdomain.

**Comment by Davi-He** [comment]
Yes, I want to set subdomain = false, but this module can not support to set subdomain, do you have any suggestion? Thank you very much.

**Comment by Davi-He** [comment]
> Hi @Davi-He do you mean to run it without a subdomain? It runs by default on a subdomain. 

Yes, I want to set subdomain = false, but this module can not support to set subdomain, do you have any suggestion? Thank you very much.

**Comment by matifali** [comment]
We need to see the kasmvnc docs to see how can we server it over a subpath.

**Comment by phorcys420** [comment]
AFAIK KasmVNC's client is a noVNC fork, so it should theoretically be possible to set the `path` query argument like w/ noVNC.

https://github.com/uwu/basic-env/blob/59d7e8d9b0328ffa59654eb8de79f200a930fcd6/main.tf#L247-L260
